### PR TITLE
translation fixes for SB

### PR DESCRIPTION
**Pull request by Akurosia**
same as for ARR and HW, here a fix of all SB translations.

### CODE REVIEW

File: ui/raidboss/data/04-sb/raid/o10n.js [hunk 1]
```diff
@@ -126,275 +126,130 @@
...
-        'Crimson Wings': '紅翼の焔',
-        'Dark Wave': 'ダークウェーブ',
         'Dry Ice': 'フリージングダスト',
         'Earth Shaker': 'アースシェイカー',
         'Exaflare': 'エクサフレア',
-        'Flame Blast': 'フレイムブラスト',
-        'Flip': 'Flip', // FIXME
         'Frost Breath': 'フロストブレス',
         'Horrid Roar': 'ホリッドロア',
-        'Hot Tail': 'ヒートテイル',
```
Comment (quisquous) [inline ui/raidboss/data/04-sb/raid/o10n.js:262]: Removing all of these lines makes me a little nervous.  How can you remove Hot Tail or Time Immemorial?

File: ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js [hunk 2]
```diff
@@ -1837,39 +1832,29 @@
...
         'Markers appear': 'Marker escheinen',
         'Megaflare': 'Megaflare',
-        'Megaflare Dive': 'Megaflare-Sturz',
         'Meteor Stream': 'Meteorflug',
         'Meteor/Dive or Dive/Beam': 'Meteor/Sturzflug oder Sturzflug/Strahl',
         'Morn Afah': 'Morn Afah',
         'Nael Dive': 'Nael Sturzflug',
         'Nael Marker': 'Nael Marker',
-        'Pepperoni': 'Pepperoni', // FIXME
+        'Pepperoni': 'Salami',
```
Comment (panicstevenson) [inline ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js:1853]: 💯

**Comment by panicstevenson** [comment]
I guess this would be a nit, but somewhat to @quisquous' point, is there a way we can discern from fixes and cleanup?

File: ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js (hunk 2 above)
Comment (Akurosia) [reply ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js:1853]: in german this is the valid translation for it and when you encountered it first in t13 it makes sence.

File: ui/raidboss/data/04-sb/raid/o10n.js (hunk 1 above)
Comment (Akurosia) [reply ui/raidboss/data/04-sb/raid/o10n.js:262]: As this is normal mode, this is never used (open js and txt file for o10n and search simply for hot, except the translation part, it is never used)

File: ui/raidboss/data/04-sb/alliance/orbonne_monastery.js [hunk 3]
```diff
@@ -855,22 +824,19 @@
...
         'I see it now': 'À vous, maintenant',
-        'Icewolf': 'grêlon de glace',
         'Iron Construct': 'bâtisseur de fer',
         'Mustadio': 'Mustadio',
         'Ramza': 'Ramza',
-        'Ruination': 'Ruinance',
-        'Shield Knight': 'chevalier au bouclier',
         'Sword Knight': 'chevalier à l\'épée',
-        'The Crystalline Gaol': 'la Geôle cristalline',
+        'The Crystalline Gaol': 'Geôle cristalline',
```
Comment (quisquous) [inline ui/raidboss/data/04-sb/alliance/orbonne_monastery.js:831]: Why are you removing the `la` here? I think this article is needed to make the regex `Fermeture d(?:e|u|es) la Geôle cristalline dans` is it really `Fermeture d(?:e|u|es) Geôle cristalline dans` in game?

**Comment by VanSlanzar** [comment]
ui/raidboss/data/04-sb/eureka/eureka_hydatos.js : Could you restore French translations like this. #1404

File: ui/raidboss/data/04-sb/eureka/eureka_hydatos.js [hunk 4]
```diff
@@ -1177,11 +1172,17 @@
         'Whirling Zantetsuken': 'Sen Zantetsuken',
         'Wyvern Explosion': 'Wyverne Explosion',
       },
+      '~effectNames': {
+        'Astral Essence': 'Bras de Lumière',
+        'Umbral Essence': 'Bras de Ténèbres',
```
Comment (VanSlanzar) [inline ui/raidboss/data/04-sb/eureka/eureka_hydatos.js:1177]: Except that which is correct, but before this area did not exist.

**Comment by Akurosia** [comment]
Will resubmit seperate PRs for sroting, removing/fixing FIXME and removing/adding notused or new lines to match find_missing_translations complaints

**Pull request closed without merging**